packages/optimistic-oracle/index.js

```diff
@@ -21,33 +21,36 @@ const { getWeb3 } = require("@uma/common");
  */
 async function run({ logger, web3, pollingDelay, errorRetries, errorRetriesTimeout }) {
   try {
-    // Load unlocked web3 accounts and get the networkId.
     const [networkId] = await Promise.all([web3.eth.net.getId()]);
-
-    // Setup contract instances. NOTE that getAddress("OptimisticOracle", networkId) will resolve to null in tests.
-    const oracle = new web3.eth.Contract(getAbi("OptimisticOracle"), getAddress("OptimisticOracle", networkId));
```

Comment:
We might need to construct this contract in this runner in the future, but I envision just passing the address + ABI to the Client and letting the client construct the contract object there.